index.bs

```diff
@@ -3212,6 +3213,82 @@ opened</dfn> steps given |window|, |type| and |message|.
 
 </div>
 
+## The emulation module ## {#module-emulation}
```

Comment:
This is probably not worth bikeshedding, but I do find the name "emulation" confusing, since we're not emulating anything. Arguably this stuff could all be in the browsingContext module, since it all applies to a specific browsing context (similarly to the way it's all exposed on `Page` in Puppeteer).

Reply:
I chose the emulation name to indicate that the commands in it change some parameters that are usually determined by the browser/page/OS on its own and are not part of regular capabilities of a browsing context (that's why it can be called "emulated" in my opinion). Also, I admit that the name was influenced by CDP :) 

> Arguably this stuff could all be in the browsingContext module

Same can be said about other commands too: e.g., Input is also scoped to the browsing context. 

I am open to moving it to the browsing context domain though if there is a consensus that it is a better place.

Reply:
This is one kind of emulation, and there are many others (see the discussion for the roadmap yesterday). There will be things like geolocation, timezone, locale, user agent, and maybe network throttling later. Maybe a dedicated module for all these different emulation features would make more sense?

Reply:
Here is the full list of emulations supported by Puppeteer: https://github.com/puppeteer/puppeteer/blob/main/packages/puppeteer-core/src/common/EmulationManager.ts#L73 

Some of them might not have been mentioned in the roadmap planning as they are of a lower priority generally but I see how features like these would fit into this module. I guess there are a few questions we could discuss:

1) do features like these deserve an own module or if we should add to existing modules (e.g., potential touch emulation would go into the input module and potential feature for disabling JavaScript would go into script)?
2) if we have a separate module, what should it be called?

Reply:
Is there any mileage in putting them in the `browser` module, which currently only contains the `close` command? I suppose not, since they’re intended to only apply to one context at a time.

Reply:
Let me move it to the browsing context module then! In the end, fewer modules is probably better and we can revisit once/if new commands are being added.